stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/Timestamp.kt

```diff
@@ -50,7 +56,10 @@ public fun Timestamp(
         when (formatType) {
             TIME -> formatter.formatTime(date)
             DATE -> formatter.formatDate(date)
-            RELATIVE -> formatter.formatRelativeTime(date)
+            RELATIVE -> when (date.isWithinDurationFromNow(ONE_MINUTE_DURATION)) {
+                true -> stringResource(id = R.string.stream_compose_message_list_footnote_edited_now)
+                else -> formatter.formatRelativeTime(date)
+            }
```

Comment:
Can't we move it into `DateFormatter`? :thinking: 
As far as `DateFormatter` lives in the common UI implementation, we will be fixing both SDKs in one shot

Reply:
I was thinking about that initially, but we truncate the future specifically for EditedAt.
In case `DateFormatter.formatRelative` is used by customers for some future dates it will be a breaking change.

Reply:
Ah, you just the formatting part. Yeah, that makes sense 👍

Reply:
It shouldn't break anything our our customer implementation